frontend/src/components/attribute/attribute_group.vue

```diff
@@ -108,18 +108,22 @@
           >
           </diffgram_select>
 
-          <v-container class="d-flex align-content-center justify-start"  v-if="group.kind === 'slider' || !group.kind">
-            <v-slider
-              :data-cy="`${group.prompt}_value_slider`"
-              :min="group.min_value"
-              :max="group.max_value"
-              v-model="internal_selected"
-              :label="group.prompt"
-              :disabled="loading || view_only_mode"
-              @change="attribute_change()"
-            >
-            </v-slider>
-            <p>{{internal_selected}}</p>
+          <v-container  v-if="group.kind === 'slider' || !group.kind">
```

Comment:
Hey @anthony-sarkis , when/if have a moment could you please comment on this changes.